bundle/Resources/config/services.yaml

```diff
@@ -55,6 +55,16 @@ services:
         tags:
             - { name: console.command, command: 'ngsite:profiler:clear-cache' }
 
+    ngsite.command.test_multiprocess:
```

Comment:
Did you mean `base_multiprocess` and `BaseMultiprocessCommand`, rather than `test` ?

Reply:
:+1: